core/src/components/badge/badge.ionic.scss

```diff
@@ -20,6 +18,32 @@
   font-weight: globals.$ion-font-weight-medium;
 }
 
+// Bold Badge
+// --------------------------------------------------
+
+:host(.badge-bold) {
+  --background: #{globals.ion-color(primary, base)};
+  --color: #{globals.ion-color(primary, contrast)};
+}
+
+:host(.badge-bold.ion-color) {
+  background: globals.current-color(base);
+  color: globals.current-color(contrast);
+}
+
+// Subtle Badge
+// --------------------------------------------------
+
+:host(.badge-subtle) {
+  --background: #{globals.ion-color(primary, base, $subtle: true)};
```

Comment:
Do we need to pass `$subtle` or can it just be `true`?

Reply:
The function accepts a few values before `$subtle`:

```scss
@function ion-color($name, $variation, $alpha: null, $rgb: null, $subtle: false) {
```

So you need to pass `$subtle` or you would have to change it to this:

```scss
--background: #{globals.ion-color(primary, base, null, null, true)};
```

I thought it was more clear what the code is doing by passing the variable.